File: Application/Services/Operations/BenchBudgetService/Dtos/ServiceDto.cs
using System;
using System.Collections.Generic;
using Application.Services.Operations.Authentication.Dtos;

namespace Application.Services.Operations.BenchBudgetService.Dtos
{
    public class ServiceDto
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public MyUserDto User { get; set; }
        public string Comments { get; set; }
        public DateTime IsAuthorized { get; set; }
        public DateTime Finished { get; set; }
        public List<RepairDto> Repairs { get; set; }
    }
}

File: Views/src/components/bench-budget-service/dto/Service-dto.ts
import { MyUser } from "src/components/authentication/dto/my-user";
import { RepairDto } from "./repair-dto";

export class ServiceDto
{
    id:number;
    userId:number;
    user:MyUser;
    comments:string;
    isAuthorized:Date;
    finished:Date;
    repairs:RepairDto[];
}
